src/components/FieldGuide.js

```diff
@@ -0,0 +1,297 @@
+import FieldGuideItemRow from './FieldGuideItemRow'
+import Icon from 'react-native-vector-icons/FontAwesome'
+import { addIndex, isEmpty, map } from 'ramda'
+
+const MAX_HEIGHT = Dimensions.get('window').height * .6
+const MIN_HEIGHT = 33
+
+export class FieldGuide extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      selectedItem: {},
+      heightAnim: new Animated.Value(0),
+      height: 0,
+      headerHeight: 0
+    }
+  }
+
+  componentWillMount() {
+     this._panResponder = PanResponder.create({
+      onStartShouldSetPanResponder: () => true,
+      onStartShouldSetPanResponderCapture: () => true,
+      onMoveShouldSetPanResponder: () => true,
+      onMoveShouldSetPanResponderCapture: () => true,
+
+      onPanResponderGrant: () => {
+        this.state.heightAnim.setOffset(this.state.heightAnim._value);
+        this.state.heightAnim.setValue(0);
+      },
+      onPanResponderMove: (e, gestureState) => {
+        const newVal = gestureState.dy * -1
+
+        Animated.event([
+            null, {dy: this.state.heightAnim}
+        ])(e, { dy: newVal });
+      },
+      onPanResponderTerminationRequest: () => true,
+      onPanResponderRelease: () => {
+        this.state.heightAnim.flattenOffset();
+
+        if (this.state.heightAnim._value < 20) {
+          this.close()
+        } else if (this.state.heightAnim._value < MIN_HEIGHT) {
+          Animated.timing(this.state.heightAnim, {
+            toValue: MIN_HEIGHT,
+            easing: Easing.out(Easing.ease),
+            duration: 100
+          }).start()
+        } else if (this.state.heightAnim._value > this.state.height) {
+          Animated.timing(this.state.heightAnim, {
+            toValue: this.state.height,
+            easing: Easing.out(Easing.ease),
+            duration: 100
+          }).start()
+        }
+      },
+    })
+  }
+
+  componentWillReceiveProps(nextProps){
+    if ((this.props.isVisible !== nextProps.isVisible) && (nextProps.isVisible === true)){
+      this.open()
+    }
+  }
+
+  open() {
+    this.state.heightAnim.setValue(0)
+    this.animateHeight(150)
+  }
+
+  close() {
+    this.animateHeight(0, 200)
+    setTimeout(()=> {
+      this.setState({selectedItem: {}, headerHeight: 0})
+      this.props.onClose()
+    }, 200)
+  }
+
+  openDetail(item) {
+    this.setState({selectedItem: item})
+    this.animateHeight(150)
+  }
+
+  closeDetail() {
+    this.setState({selectedItem: {}, headerHeight: 0})
+    this.animateHeight(150)
+  }
+
+  animateHeight(toHeight, duration=300) {
+    Animated.timing(
+      this.state.heightAnim,
+      {
+        toValue: toHeight,
+        easing: Easing.linear,
+        duration: duration,
+      }
+    ).start()
+  }
+
+  setHeight(height) {
+    const newHeight = height + this.state.headerHeight
+    this.setState({height: newHeight})
+    this.animateHeight(newHeight < MAX_HEIGHT ? newHeight : MAX_HEIGHT)
```

Comment:
It should display all items (and automatically become higher if there are more items), but to the max of 60% of the screen (which is admittedly arbitrary, happy to change it if you think it makes sense).  It should be displaying all items,however.  Is that not what you're seeing?  (I just added some more in the field guide)

![image](https://cloud.githubusercontent.com/assets/1033477/25397133/cbd51f4e-29ac-11e7-8aee-a602a4dcba69.png)